source/a_releasenotes.rst

```diff
@@ -0,0 +1,65 @@
+.. raw:: html
+
+    <style> .blue {color:#4a86e8} </style>
+
+.. role:: blue
+
+====================
+Latest Release Notes
+====================
+
+We’re happy to announce the newest release of Lizard Portal, Lizard Backend and Lizard Catalogue. If you have questions about this release or if you’re interested in features please contact us via info@lizard.net  
+
+Frontend
+========
+
+Lizard Client
+-------------
+
+* Reorganisation of the omnibox
+
+    * Multiple legends below each other
+    * Name of the raster and organisation added to diagrams and legends
+
+.. image:: /images/a_releasenotes_01.jpg
+
+Catalogue
+---------
+
+The Lizard Catalogue offers insight in the data layers that are available for your organisation. There is an extensive search option to make the layers easily accesible. Every data layer will show available metadata. From the Catalogue you have the option of opening the data layers via the API or via the Lizard portal.
+
+The Catalogue can be reached via this url: https://demo.lizard.net/catalogue/
+
+.. image:: /images/a_releasenotes_02.jpg
+
+Backend
+=======
+
+API
+---
+
+* added api/v4/wmslayers/
```

Comment:
Hoofdletter aan het begin.

Omdat er vaak toch bij een van de items wel een tweede zin nodig is, hou ik voor mezelf aan bullet points altijd met een hoofdletter te beginnen en met een punt te laten eindigen.